ml_git/ml_git_message.py

```diff
@@ -218,6 +218,7 @@
     'ERROR_ENTITY_NEEDS_CATATEGORY': 'You must place at least one category in the entity .spec file',
     'ERROR_INVALID_DATASET_SPEC': 'Error: invalid dataset spec (Missing name). It should look something like this:\n%s',
     'ERROR_REPOSITORY_NOT_FOUND': 'No repositories found, verify your configurations!',
+    'ERROR_REPOSITORY_NOT_FOUND_FOR_ENTITY': 'No repositories found for %s, verify your configurations!',
```

Comment:
```suggestion
    'WARN_REPOSITORY_NOT_FOUND_FOR_ENTITY': 'No repositories found for %s, verify your configurations!',
```

Please, move this constant to the warning block below.

Reply:
Changed in 4616d37